frontend/resourceadm/pages/AboutResourcePage/AboutResourcePage.tsx

```diff
@@ -163,7 +163,7 @@ export const AboutResourcePage = ({
         />
         <ResourceSwitchInput
           label={t('resourceadm.about_resource_delegable_label')}
-          description={t('resourceadm.about_resource_delegable_text')}
+          description=''
```

Comment:
Is this supposed to send in an empty string?

Reply:
fixed; made the prop optional instead of sending empty string as prop